jinja2/ext.py

```diff
@@ -434,6 +446,57 @@ class AutoEscapeExtension(Extension):
     pass
 
 
+class DebugExtension(Extension):
+    """
+    A {% debug %} tag that dumps the available variables, filters and tests.
+    This is roughly equivalent to the Django Template Language {% debug %} tag,
```

Comment:
No need to mention Django here, this won't mean anything if you haven't used the Django debug tag, and it's not really equivalent anyway.

Reply:
Ack.